client/src/components/BoulderCard.jsx

```diff
@@ -17,15 +18,15 @@ const BoulderCard = ({
   img_start,
   setter,
   weighting,
+  detailedMode
```

Comment:
Instead of all these props separatly, you could pass one object "Boulder" as a prop and then, you destructure it below similar to this:
`const {id, name, sector, level, ..., detailedMode} = Boulder;`

Reply:
I have changed it accordingly.